http/oidc/src/main/java/org/wildfly/security/http/oidc/Oidc.java

```diff
@@ -445,4 +452,14 @@ protected static boolean checkCachedAccountMatchesRequest(OidcAccount account, O
         return true;
     }
 
+    public static String getCryptographicValue(final String src) {
```

Comment:
Does this need to be public?

Reply:
Yes it is referenced by TokenValidator and OidcRequestAuthenticator

Reply:
Could it be protected instead?

Reply:
If I move it into OidcHttpFacade it could be made pubic and accessible to TokenValidator and OidcRequestAuthenticator.

Would you prefer that?

Reply:
It's fine to have utility methods in Oidc.java but we typically make them protected where it makes sense to do so.

Reply:
done